cmake/modules/RootMacros.cmake

```diff
@@ -873,29 +1022,29 @@ function(ROOT_ADD_INCLUDE_DIRECTORIES library)
       # get a redefinition error.
       # We should remove these lines when the fallback include is removed. Then
       # we will need a module.modulemap file per `inc` directory.
-      target_include_directories(${library} BEFORE PRIVATE ${CMAKE_BINARY_DIR}/include)
+      # target_include_directories(${library} PRIVATE ${CMAKE_BINARY_DIR}/include)
```

Comment:
@vgvassilev can you take a look here please? (@linev you want to leave commented?)

Reply:
I placed ` ${CMAKE_BINARY_DIR}/include` in the beginning of the list anyway. Here:
https://github.com/root-project/root/pull/5181/files#diff-aa53cf21618fabeb12002f796b8cf2fbR291
Idea to remove it at some point - if we can fix problem with rootcling